src/util.ts

```diff
@@ -20,3 +20,12 @@ export function getSummary(result: unknown): string {
 
   return parsedResult.success ? ` - ${parsedResult.data.summary}` : '';
 }
+
+export function isJsonString(hardware: string): boolean {
+  try {
+    JSON.parse(hardware);
+  } catch (e) {
+    return false;
```

Comment:
@phracek Maybe add some logging?

Reply:
Wouldn't it be better not to catch the exception and let the action fail? If somebody is trying to set hardware requirements, it's probably required for the test to run successfully.

Reply:
This is fixed by commit https://github.com/sclorg/testing-farm-as-github-action/pull/146/commits/cb3b7bc435c80b0563d52914ba7235c4985b6d1f. 
Function isJsonString is not needed now.